src/test/java/org/jenkinsci/plugins/credentialsbinding/impl/BindingStepTest.java

```diff
@@ -262,6 +262,31 @@ private static FilePath tempDir(FilePath ws) {
             }
         });
     }
+    
+    @Issue("JENKINS-24805")
+    @Test public void maskingSimilarSecrets() {
```

Comment:
…while there is no test reproducing the _reported_ bug, affecting only freestyle projects.